integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -537,11 +535,9 @@ def export(self, **kwargs):
 
                 update_payload[key] = item
             for key, item in items_to_delete.items():
-                self._sanitize_and_set_item_metadata(item, key_to_content_metadata_mapping[key], IC_DELETE_ACTION)
-                # Sanity check
-                item.enterprise_customer_catalog_uuid = enterprise_customer_catalog.uuid
+                metadata = self._apply_delete_transformation(item.channel_metadata)
```

Comment:
Because we can't request metadata of content that isn't in the customer's catalog, we have to make do with the audit's channel metadata and leave it up to the individual channel as to what needs sanitizing